modulefiles/gfsutils_container.intel.lua

```diff
@@ -0,0 +1,17 @@
+help([[
+Build environment for GFS utilities in a container 
+]])
+
+prepend_path("MODULEPATH", "/opt/spack-stack/spack-stack-1.8.0/envs/unified-env/install/modulefiles/Core")
+
+local stack_intel_ver=os.getenv("stack_intel_ver") or "2021.10.0"
+local stack_impi_ver=os.getenv("stack_impi_ver") or "2021.12.1"
+local cmake_ver=os.getenv("cmake_ver") or "3.27.9"
+
+load(pathJoin("stack-intel", stack_intel_ver))
+load(pathJoin("stack-intel-oneapi-mpi", stack_impi_ver))
+load(pathJoin("cmake", cmake_ver))
+
+load("gfsutils_generic")
```

Comment:
Why would the container not load the `gfsutils_common.lua` modulefile?

Reply:
The version numbers are not the same in the container, since it is based on spack-stack 1.8.0.

Reply:
Understood. Thanks for explaining.
@DavidBurrows-NCO Please create an issue to replace this and consolidate w/ `gfsutils_common.lua` and remove this file when spack-stack is upgraded to 1.8.0 so that we can revisit it then. 
We can accept this for now, but please we cannot afford to maintain multiple copies of modulefiles for different spack-stack versions.  That would be unmanageable for maintenance teams.

Reply:
@aerorahul @mark-a-potts Issue #88 created

Reply:
@aerorahul Container is now using gfsutils_common. Thanks @mark-a-potts